Essentials/src/main/java/com/earth2me/essentials/EssentialsPlayerListener.java

```diff
@@ -346,7 +346,7 @@ public void run() {
                         .replace("{UPTIME}", DateUtil.formatDateDiff(ManagementFactory.getRuntimeMXBean().getStartTime()))
                         .replace("{PREFIX}", FormatUtil.replaceFormat(ess.getPermissionsHandler().getPrefix(player)))
                         .replace("{SUFFIX}", FormatUtil.replaceFormat(ess.getPermissionsHandler().getSuffix(player)))
-                        .replace("{OLDUSERNAME}", lastAccountName);
+                        .replace("{OLDUSERNAME}", lastAccountName == null ? "" : lastAccountName);
```

Comment:
Does this work? I did `newUsername ? lastAccountName : ""`, but I suppose that's checked by what message it's doing. Wouldn't this plop the current username if it hadn't changed instead of nothing?

Reply:
> Wouldn't this plop the current username if it hadn't changed instead of nothing?
This is intended, not really worth separating them since we don't encourage the use of it anyway.